guide/README.md

```diff
@@ -455,6 +455,56 @@ used to bump the `Mi 4` note up by one octave and then to bump the
 | | | For example: Transposing C-D-E-F by 4 (fifth) will give us G-A-B-C
 | | | To transpose an octave: shift by the mode length (7 in major scales) up or down. |
 
+#### <a name="Set Key"></a>3.2.5 Set Key
+
+![alt tag](./setkey1.svg "scalar transposition")
+
+
+The Set key block is used to set the key and mode, eg C Major. 
+It allows you to change both the mode and key of how solfege is mapped to the notes.
+
+
+| Standard Notation for Set Key |
+| --- |
+| ![alt tag](./setkey2.png "Standard Notation Set Key") |
+
+[RUN LIVE](https://musicblocks.sugarlabs.org/index.html?id=1662103714150464&run=True)
+
+In set key , mode determines the pattern of notes and key determines where the pattern starts. In the following table it is shown how notes and modes pattern changes with changing value of set key value.
+
+| Set Key Value | Notes Pattern | Modes Pattern |
+| --- | --- | --- |
+| C Major | C, D, E, F, G, A, B, C | 2 2 1 2 2 2 1 |
+| F# Major | F#, G#, A#, B, C#, D#, F, F# | 2 2 1 2 2 2 1 |
+| A♭ Major | Ab, Bb, C, Db, Eb, F, G, Ab | 2 2 1 2 2 2 1 |
+| E Major | E, F#, G#, A, B, C#, D#, E | 2 2 1 2 2 2 1 |
+| A# Major | Bb, C, D, Eb, F, G, A, Bb | 2 2 1 2 2 2 1 |
+| --- | --- | --- |
+| C Dorian | C, D, Eb, F, G, A, Bb, C | 2 1 2 2 2 1 2 |
+| F# Dorian | F#, G#, A, B, C#, D#, E, F# | 2 1 2 2 2 1 2 |
+| A♭ Dorian | G#, A#, B, C#, D#, F, F#, G# | 2 1 2 2 2 1 2 |
+| E Dorian | E, F#, G, A, B, C#, D, E | 2 1 2 2 2 1 2 |
+| A# Dorian | Bb, C, Db, Eb, F, G, Ab, Bb | 2 1 2 2 2 1 2 |
+| --- | --- | --- |
+| C Phrygian | C, Db, Eb, F, G, Ab, Bb, C | 1 2 2 2 1 2 2 |
+| F# Phrygian | F#, G, A, B, C#, D, E, F# | 1 2 2 2 1 2 2 |
+| A♭ Phrygian | G#, A, B, C#, D#, E, F#, G# | 1 2 2 2 1 2 2 |
+| E Phrygian | E, F, G, A, B, C, D, E | 1 2 2 2 1 2 2 |
+| A# Phrygian | Bb, Cb, Db, Eb, F, Gb, Ab, Bb | 1 2 2 2 1 2 2 |
+
+**Set Key & Movable Do**
```

Comment:
I'd explain how set key impacts scalar step first.

In the solfege explanation, use a key other than C. Don't bother with the "Modes include.... etc."
You should say that set key only impact the mapping of solfege when the "movable Do* block is set to True.